OpenKeychain/src/main/java/org/sufficientlysecure/keychain/util/orbot/OrbotStatusReceiver.java

```diff
@@ -0,0 +1,78 @@
+package org.sufficientlysecure.keychain.util.orbot;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import org.sufficientlysecure.keychain.util.Log;
+
+/**
+ * Created by vanitas on 11.05.16.
```

Comment:
remove the "created by" line, instead add a license header like in other files with your name